### ISSUE TITLE
CVE in Pillow

### ISSUE THREAD

**Issue by vanHavel** [heartexlabs/label-studio#4893]
Hi,

our security scanner (`trivy`) reports a `HIGH` severity vulnerability in the `Pillow` version (`9.3.0`) included in the `label-studio` docker image (currently `1.9.1` or `latest`). 

```
$ trivy image --security-checks vuln heartexlabs/label-studio:1.9.1

┌────────────────────┬─────────────────────┬──────────┬───────────────────┬────────────────┬──────────────────────────────────────────────────────────┐
│      Library       │    Vulnerability    │ Severity │ Installed Version │ Fixed Version  │                          Title                           │
├────────────────────┼─────────────────────┼──────────┼───────────────────┼────────────────┼──────────────────────────────────────────────────────────┤
│ Pillow (METADATA)  │ CVE-2023-4863       │ HIGH     │ 9.3.0             │ 10.0.1         │ Heap buffer overflow in WebP Codec                       │
│                    │                     │          │                   │                │ https://avd.aquasec.com/nvd/cve-2023-4863                │
│                    ├─────────────────────┤          │                   │                ├──────────────────────────────────────────────────────────┤
│                    │ GHSA-56pw-mpj4-fxww │          │                   │                │ Bundled libwebp in Pillow vulnerable                     │
│                    │                     │          │                   │                │ https://github.com/advisories/GHSA-56pw-mpj4-fxww        │
├────────────────────┼─────────────────────┼──────────┼───────────────────┼────────────────┼──────────────────────────────────────────────────────────┤

```

The vulnerability is fixed in `Pillow >= 10.0.1`. 

Is it possible to upgrade `Pillow` to fix this CVE?

**Comment by hogepodge** [comment]
Thanks @vanHavel, I'll bring this to our security team to talk about next steps.

**Comment by jombooth** [comment]
Fixed in [1.9.1.post0](https://github.com/HumanSignal/label-studio/releases/tag/1.9.1.post0) - thank you for the report!
